README.md

```diff
@@ -43,17 +40,94 @@ Using the command line:
+
+The **master** branch contains the code generated from Hydra/Hyak.
+  - Hydra/Hyak is Azure's legacy code generation technology.
+  - This can still be used to generate client libraries, but the project is not being advanced in favor of AutoRest. Your team should move to AutoRest and Swagger as soon as possible.
+
+### Standard Process
+
+ 1. Create fork of [Azure REST API Specs](https://github.com/azure/azure-rest-api-specs)
+ 2. Create fork of [Azure SDK for .NET](https://github.com/azure/azure-sdk-for-net)
+ 3. Create your Swagger specification for your HTTP API. For more information see 
+ [Introduction to Swagger - The World's Most Popular Framework for APIs](http://swagger.io)
+ 4. Install the latest version of AutoRest and use it to generate your C# client. For more info on getting started with AutoRest, 
+ see the [AutoRest repository](https://github.com/Azure/autorest)
+ 5. Create a branch in your fork of Azure SDK for .NET and add your newly generated code to your project. If you don't have
+ a project in the SDK yet, look at some of the existing projects and build one like the others. 
+ 6. **MANDATORY**: Add or update tests for the newly generated code.
+ 7. Once added to the Azure SDK for .NET, Build your local package using command "msbuild build.proj /t:build;package /p:scope=YourService" 
+ (Note, 'YourService' comes from the sub folder under <sdk-repo-root>\src, for example: "ResourceManagement\Compute")
+ 8. Bump up the package version in YourService.nuget.proj
+ 9. Use this local Package for your Powershell development
+ 10. Create 2 Pull Requests and send an email to [azsdkcode@microsoft.com](mailto:azsdkcode@microsoft.com)
+    - A Pull Request of your spec changes against **master** branch of the [Azure REST API Specs](https://github.com/azure/azure-rest-api-specs)
+    - A Pull request of your Azure SDK for .NET changes against **master** branch of the [Azure SDK for .NET](https://github.com/azure/azure-sdk-for-net)
+ 11. Both the pull requests will be reviewed and merged by the Azure SDK team
+
+### Code Review Process
+
+Before a pull request will be considered by the Azure SDK team, the following requirements must be met:
+
+- Prior to issuing the pull request:
+  - All code must have completed any necessary legal signoff for being publically viewable (Patent review, JSR review, etc.)
+  - The changes cannot break any existing functional /unit tests that are part of the central repository.
+    - This includes all tests, even those not associated with the given feature area.
+  - Code submitted must have basic unit test coverage, and have all the unit tests pass. Testing is the full responsibility of the service team
+    - Functional tests are encouraged, and provide teams with a way to mitigate regressions caused by other code contributions.
+  - Code should be commented.
+  - Code should be fully code reviewed.
+  - Code should be able to merge into the dev branch being targeted.
```

Comment:
>Code should be able to merge into the dev branch being targeted. [](start = 4, length = 64)

Please rephrase this to indicate that we want partners to have a PR without merge conflicts.